src/EnergyPlus/VentilatedSlab.hh

```diff
@@ -65,6 +65,56 @@ struct EnergyPlusData;
 
 namespace VentilatedSlab {
 
+    enum class HeatingCoilType {
+        Invalid = -1,
+        Electric,
+        Gas,
+        Water,
+        Steam,
+        Num
+    };
+
+    enum class CoolingCoilType {
+        Invalid = -1,
+        WaterCooling,
+        DetailedCooling,
+        HXAssisted,
+        Num
+    };
+
+    // Parameters for outside air control types:
+    enum class OutsideAirControlType
+    {
+        Invalid = -1,
+        VariablePercent,
+        FixedTemperature,
+        FixedOAControl,
+        Num
+    };
+    
+    enum class CoilType {
+        Invalid = -1,
+        None,
+        Heating,
+        Cooling,
+        Both,
+        Num        
+    };
+
+    //  Control Types
+    enum class ControlType
+    {
+        Invalid = -1,
```

Comment:
I think these should be spelled out, maybe not completely but more, e.g., `MeanAirTemp`, `OperativeTemp`, `OutdoorDryBulbTemp`, etc.

Reply:
Will change.